index.html

```diff
@@ -432,50 +444,62 @@
       <div id="nav-popout"> 
 
       </div>
-      
+
       <div class="cardContainer" id="cardContainer">
 
       </div>
 
+      <div id="metamaskWarning">
+        <img id="metamaskImg" src="./metamask.png">
+        <h3><a href="https://metamask.io/" target="_blank">Metamask Extension</a> required to use Exokit Store.</h3>
+      </div>
+
       <div class=multibutton>
         <a class="button first last" id="enter-xr-button" style="display: none;">Enter XR</a>
         <a class="button first last" id="no-xr-button" disabled style="display: none;">No XR</a>
       </div>
     </div>
     <script>
 (async () => {
+
+  const metamaskWarning = document.getElementById("metamaskWarning");
+
   const contract = await new Promise((accept, reject) => {
     window.addEventListener('load', async () => {
+
+      const accessContract = async () => {
```

Comment:
This code flow is strange. This is an async function which is not called asyncly, but instead accepts another promise. That's pretty confusing.

Can we instead straight line code flow?

Reply:
fixed